plugins/modules/sensor_download.py

```diff
@@ -65,6 +66,13 @@
     hash: "1234567890123456789012345678901234567890123456789012345678901234"
     dest: "/tmp/windows"
     name: falcon-sensor.exe
+
+- name: Download the Falcon Sensor Installer to a temporary directory and set permissions
+  crowdstrike.falcon.sensor_download:
+    hash: "1234567890123456789012345678901234567890123456789012345678901234"
+    mode: "0755"
```

Comment:
@carlosmmatos One last question. Are you sure about these permissions? Using 0644 should be sufficient.

Reply:
It was an example and clearly I was not thinking right last night.. I will update that to 0644